shell/platform/darwin/ios/framework/Source/FlutterPlatformPluginTest.mm

```diff
@@ -17,9 +17,27 @@ @interface FlutterPlatformPluginTest : XCTestCase
 
 @interface FlutterPlatformPlugin ()
 - (BOOL)isLiveTextInputAvailable;
+- (void)handleSearchWebWithSelection:(NSString*)selection;
 @end
 
 @implementation FlutterPlatformPluginTest
+- (void)testSearchWebInvokedCorrectly {
+  FlutterEngine* engine = [[[FlutterEngine alloc] initWithName:@"test" project:nil] autorelease];
+  std::unique_ptr<fml::WeakPtrFactory<FlutterEngine>> _weakFactory =
+      std::make_unique<fml::WeakPtrFactory<FlutterEngine>>(engine);
+  XCTestExpectation* invokeExpectation = [self expectationWithDescription:@"isSearchWebInitiated"];
+  FlutterPlatformPlugin* plugin =
+      [[[FlutterPlatformPlugin alloc] initWithEngine:_weakFactory->GetWeakPtr()] autorelease];
+  FlutterPlatformPlugin* mockPlugin = OCMPartialMock(plugin);
+  FlutterMethodCall* methodCall = [FlutterMethodCall methodCallWithMethodName:@"SearchWeb.initiate"
+                                                                    arguments:@"Test"];
```

Comment:
Nit: Maybe you could also try testing that a string with some characters that need to be escaped get escaped?  If you think it's useful.